requirements.txt

```diff
@@ -1 +1,3 @@
-premailer
\ No newline at end of file
+premailer
+simplejson
+sendgrid
```

Comment:
Can you please add external python dependencies in the manifest too?

Reply:
done!